InvenTree/InvenTree/settings.py

```diff
@@ -25,6 +25,10 @@
 import yaml
 from django.utils.translation import gettext_lazy as _
 from django.contrib.messages import constants as messages
+from psycopg2.extensions import (
+    ISOLATION_LEVEL_READ_COMMITTED,
+    ISOLATION_LEVEL_SERIALIZABLE,
+)
```

Comment:
I suggest putting the imports into line 499ff to make sure it does not require the pgsql bindings if not using postgres

Reply:
Yeah, that would work as well. I went a slightly different way though.